### ISSUE TITLE
where is fit3Dline_nonan ?

### ISSUE THREAD

**Issue by alexpono** [turbulencelyon/4d-ptv#10]
Hello,
the function fit3Dline is calling the function fit3Dline_nonan,
But I do not see it in the folder Center2Rays ?
Best,
Pono

**Comment by jsalort** [comment]
There is a `fit3Dline_nan`. Maybe the function was renamed, and the old name has not been changed accordingly?
@ddumont-2607 ?

**Comment by ddumont-2607** [comment]
Indeed, having all detected particle in the convex hull is very unusual and in this case a function was missing. I corrected this bug in the last commit just now. Please @alexpono try it now and tell us if this issue is fixed or not.
